renovate.json

```diff
@@ -1,5 +1,10 @@
 {
   "extends": [
     "config:base"
-  ]
+  ],
+  "rangeStrategy": false,
```

Comment:
Thanks for updating this. It looks like ["rangeStrategy"](https://docs.renovatebot.com/configuration-options/#rangestrategy)s shouldn't be a boolean but the type of behavior. Should we use "replace"?

Reply:
Eugh, you're right - I got half way between two different config settings (will update)